test/css/testing.mjs

```diff
@@ -4,6 +4,7 @@ const configs = [
   ['test/default-css', {}],
   ['test/process-css', { include: [/App\.css/] }],
   ['test/process-module', { include: [/App\.module\.css/] }],
+  ['test/process-inline', { include: [/App\.module\.css/] }],
```

Comment:
These have the same config so it's better to allow passing an array as the first argument I think